library/aik099/PHPUnit/BrowserConfiguration/BrowserConfiguration.php

```
@@ -356,6 +357,36 @@ public function getBaseUrl()
 	}
 
 	/**
+	 * Set the mink driver class to use.
+	 *
+	 * @param string $driver_class The mink driver class to use.
+	 *
+	 * @return self
+	 *
+	 * @throws \InvalidArgumentException When mink driver is not a valid class.
+	 */
+	public function setMinkDriverClass($driver_class)
+	{
+		if ( ! is_string($driver_class) ) {
+			throw new \InvalidArgumentException('The mink driver class must be a string');
+		}
+
+		$this->parameters['minkDriverClass'] = $driver_class;
```

If I remember correctly, then it's considered good practice to fail early and closer to the issue place so that exception trace is more meaningful.

For example I validate all browser configuration parameters right in setter method. See no problems if you do same here.